components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.experiments.debug
+
+import android.app.Activity
+import android.os.Bundle
+import android.support.annotation.VisibleForTesting
+import mozilla.components.service.experiments.Experiments
+import mozilla.components.support.base.log.logger.Logger
+
+class ExperimentsDebugActivity : Activity() {
+    private val logger = Logger(this.javaClass.canonicalName)
+
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    internal var testExperimentsUpdated = false
+
+    companion object {
+        // This is a list of the currently accepted commands
+        const val UPDATE_EXPERIMENTS_EXTRA_KEY = "updateExperiments"
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        logger.debug("ExperimentsDebugActivity onCreate")
```

Comment:
nit: this doesn't need `EperimentsDEbugActivity` since this should be in the Logger name, as Georg suggests.